src/api/api.jl

```diff
@@ -1,16 +1,46 @@
 module API
 
-import Libdl
+using Libdl
```

Comment:
```suggestion
using Libdl: dlopen, dlclose, dlpath, RTLD_LAZY, RTLD_NODELETE
```
To be consistent with the other `using` statements?